presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
+* Added support for memozing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
+* Enforce NOT NULL constraint by checking values for NOT NULL columns.
+* NodeStatusService implementation is now expected to validate & check if a node is allowed to register with DiscoveryNodeManager based on node.location instead of node.id configuration parameter.
+
+JDBC Connector Changes
+______________________
+* JDBC connector allows user to pass username and password from CLI and other clients through use of extra credentials feature.
+
+Hive Changes
+____________
+* Support Cache Quota when using Alluxio cache
+* Add configuration property cache.alluxio.eviction-policy to set eviction policy for Alluxio caching.
+
+Verifier Changes
+________________
+* Increase verifier retry and determinism analysis limits.
```

Comment:
I think we can skip this

Reply:
Verifier is more technical in nature and I think it's relevant.  If one uses verifier, they'll be interested in this change.